include/NovelRT/Maths/GeoMatrix4x4F.h

```diff
@@ -22,139 +25,726 @@ namespace NovelRT::Maths
 
+        /**
+         * @brief Constructs a matrix containing the product of this and another GeoMatrix4x4F.
+         *
+         * @details
+         * An example of a matrix that would be constructed from multiplying two matrices: \f[
+         *      \begin{bmatrix}
+         *      1 & 7 & 6 & 5\\
+         *      2 & 1 & 4 & 3\\
+         *      3 & 5 & 1 & 2\\
+         *      4 & 6 & 7 & 1
+         *      \end{bmatrix}
+         *      \cdot
+         *      \begin{bmatrix}
+         *      4 & 7 & 2 & 5\\
+         *      5 & 1 & 8 & 4\\
+         *      8 & 3 & 2 & 2\\
+         *      6 & 7 & 6 & 8
+         *      \end{bmatrix}
+         *      =
+         *      \begin{bmatrix}
+         *      1*4+7*5+6*8+5*6 & 1*7+7*1+6*3+5*7 & 1*2+7*8+6*2+5*6 & 1*5+7*4+6*2+5*8\\
+         *      2*4+1*5+4*8+3*6 & 2*7+1*1+4*3+3*7 & 2*2+1*8+4*2+3*6 & 2*5+1*4+4*2+3*8\\
+         *      3*4+5*5+1*8+2*6 & 3*7+5*1+1*3+2*7 & 3*2+5*8+1*2+2*6 & 3*5+5*4+1*2+2*8\\
+         *      4*4+6*5+7*8+1*6 & 4*7+6*1+7*3+1*7 & 4*2+6*8+7*2+1*6 & 4*5+6*4+7*2+1*8
+         *      \end{bmatrix}
+         *      \rightarrow
+         *      \begin{bmatrix}
+         *      117 & 67 & 100 & 85\\
+         *      63 & 48 & 38 & 46\\
+         *      57 & 43 & 60 & 53\\
+         *      108 & 62 & 76 & 66
+         *      \end{bmatrix}
+         * \f]
+         * @param other The other matrix who's columns are multiplied and summed up by this matrix's rows.
+         * @return a matrix who's components consist of the product of this and another GeoMatrix4x4F.
+         */
         inline GeoMatrix4x4F operator*(GeoMatrix4x4F other) const noexcept
         {
             return GeoMatrix4x4F(*reinterpret_cast<const glm::mat4*>(this) *
                                  *reinterpret_cast<const glm::mat4*>(&other));
         }
 
+        /**
+         * @brief Constructs a matrix containing the entrywise sum of this and another GeoMatrix4x4F and applies it to
+         * this instance.
+         * @details
+         * An example of a matrix that would be constructed from an entrywise sum: \f[
+         *      \begin{bmatrix}
+         *      1 & 7 & 6 & 5\\
+         *      2 & 1 & 4 & 3\\
+         *      3 & 5 & 1 & 2\\
+         *      4 & 6 & 7 & 1
+         *      \end{bmatrix}
+         *      +
+         *      \begin{bmatrix}
+         *      4 & 7 & 2 & 5\\
+         *      5 & 1 & 8 & 4\\
+         *      8 & 3 & 2 & 2\\
+         *      6 & 7 & 6 & 8
+         *      \end{bmatrix}
+         *      =
+         *      \begin{bmatrix}
+         *      1+4 & 7+7 & 6+2 & 5+5\\
+         *      2+5 & 1+1 & 4+8 & 3+4\\
+         *      3+8 & 5+3 & 1+2 & 2+2\\
+         *      4+6 & 6+7 & 7+6 & 1+8
+         *      \end{bmatrix}
+         *      \rightarrow
+         *      \begin{bmatrix}
+         *      5 & 14 & 8 & 10\\
+         *      7 & 2 & 12 & 7\\
+         *      11 & 8 & 3 & 4\\
+         *      10 & 13 & 13 & 9
+         *      \end{bmatrix}
+         * \f]
```

Comment:
As per my other question, is this correct?

Reply:
after testing this works as expected.